src/main/java/jimmy/mcgymmy/model/food/Food.java

```diff
@@ -64,6 +64,13 @@ private boolean isValidName(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    /**
+     * @return total caloric content of food item
+     */
+    public int getCalories() {
+        return this.protein.getTotalCalories() + this.carbs.getTotalCalories() + this.fat.getTotalCalories();
```

Comment:
Might be better to abstract this.protein to getProtein(), same for carbs and fats

Reply:
Alright!